src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/GetProfileNameDialog.xaml

```diff
@@ -38,8 +39,8 @@
                 <ColumnDefinition Width="Auto"></ColumnDefinition>
                 <ColumnDefinition Width="Auto"></ColumnDefinition>
             </Grid.ColumnDefinitions>
-            <Button x:Uid="OKButton" Grid.Row="0"  Grid.Column="0" Margin="5" MinWidth="50" Padding="10,2" IsDefault="True" Click="OKButton_Click">OK</Button>
-            <Button x:Uid="CancelButton" Grid.Row="0"  Grid.Column="1" Margin="5" MinWidth="50" Padding="10,2" IsCancel="True">Cancel</Button>
+            <Button x:Uid="OKButton" Grid.Row="0"  Grid.Column="0" Margin="5" MinWidth="50" Padding="10,2" IsDefault="True" Click="OKButton_Click" Content="{x:Static Member=local:PropertyPageResources.OKBtn}" />
```

Comment:
Are there no system standard strings for these?

Reply:
I do not know. Is there some existing VS API for general-use localized strings?

Reply:
At least looking through our own code base, we have other resource declarations for OK/Cancel, but nothing that looks to be easily reusable here.

The Roslyn example you gave is in the same boat, it ends up pulling in from a Roslyn resource file declaring the string.